x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/creation.cy.ts

```diff
@@ -155,6 +145,7 @@ describe.skip('Timelines', { tags: ['@ess', '@serverless'] }, (): void => {
   });
 
   it('should save timelines as new', () => {
+    login();
     visitWithTimeRange(TIMELINES_URL);
     cy.get(ROWS).should('have.length', '0');
 
```

Comment:
line 161 I wonder why we need to `.filter(':visible')` inside the implementation of the `closeTimeline` function

Reply:
I'm not sure and removing it doesn't seem to have an impact 👍